apps/backoffice-v2/src/common/hooks/useFilter/useFilter.tsx

```diff
@@ -6,7 +6,7 @@ export const useFilter = () => {
 
   const onFilter = useCallback(
     (accessor: string) => {
-      return (values: string[]) => {
```

Comment:
I see this changes go back and forth quite often. Do we have a hard stance on it, or an ESLint rule to make it consistent throughout our repo?

Reply:
Don't we already have ESLint in our pre-commit hooks? Or is it just prettier?

Reply:
Just prettier.